lib/Thingiverse.pm

```diff
@@ -60,8 +52,19 @@ our $client_id          = 'c587f0f2ee04adbe719b';
 our $access_token       = 'b053a0798c50a84fbb80e66e51bba9c4';
```

Comment:
I like most of what you did.  I wish you hadn't called it "Cleanup" :).

So I am a little confused about whether this is ready to merge or not.